x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexing/IndexerJobStats.java

```diff
@@ -30,7 +30,8 @@
     private static ParseField NUM_PAGES = new ParseField("pages_processed");
     private static ParseField NUM_INPUT_DOCUMENTS = new ParseField("documents_processed");
     // BWC for RollupJobStats
-    private static ParseField NUM_OUTPUT_DOCUMENTS = new ParseField("documents_indexed").withDeprecation("rollups_indexed");
+    private static String ROLLUP_BWC_NUM_OUTPUT_DOCUMENTS = "rollups_indexed";
+    private static ParseField NUM_OUTPUT_DOCUMENTS = new ParseField("documents_indexed").withDeprecation(ROLLUP_BWC_NUM_OUTPUT_DOCUMENTS);
```

Comment:
note: you can pass the deprecated name as a second argument of the constructor here and this will only create one instance instead of creating an instance and then using it to create a second instance with `withDeprecation()`. Not a big thing at all and either way is fine but wanted to point it out as its how we have done this elsewhere